src/configurations/adknowledgeportal/synapseConfigs/projects.ts

```diff
@@ -68,15 +67,35 @@ export const projectsDetailsPageConfiguration: DetailsPageProps = {
       },
     },
     {
-      name: 'CardContainerLogic',
-      columnName: 'Grant Number',
+      name: 'ToggleSynapseObjects',
       title: 'Experimental Models',
       showTitleSeperator: false,
-      tableSqlKeys: ['grant'],
-      props: {
-        sql: experimentalModelsSql,
-        ...experimentalToolsCardConfiguration,
+      standalone: true,
+      toggleConfigs: {
+        icon1: 'table',
+        config1: {
+          name: 'StandaloneQueryWrapper',
+          props: {
+            sql: experimentalModelsSql,
+            rgbIndex,
+            sqlOperator: '=',
+            columnLinks: experimentalDetailsTableConfigurationColumnLinks,
+          },
+          columnName: 'Grant Number',
+          tableSqlKeys: ['grant'],
+        },
+        icon2: 'entityview',
```

Comment:
Need to update this icon before merge (to 'cards', which will be available in the next SRC release)